cfme/tests/v2v/test_v2v_ansible.py

```diff
@@ -90,12 +90,28 @@ def _cleanup():
 
 
 @pytest.mark.parametrize(
-    "form_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
+    "mapping_data_vm_obj_single_datastore", [["nfs", "nfs", rhel7_minimal]], indirect=True
 )
-def test_migration_playbooks(request, appliance, v2v_providers, host_creds, conversion_tags,
-                             ansible_repository, form_data_vm_obj_single_datastore):
-    """Test for migrating vms with pre and post playbooks"""
-    creds = credentials[v2v_providers.vmware_provider.data.templates.get("rhel7_minimal").creds]
+def test_migration_playbooks(request, appliance, source_provider, provider,
+                             ansible_repository, mapping_data_vm_obj_single_datastore):
+    """
+    Test for migrating vms with pre and post playbooks
+
+    Polarion:
+        assignee: sshveta
+        caseimportance: medium
+        casecomponent: V2V
+        initialEstimate: 1/4h
+        testSteps:
+            1. Enable embedded ansible role
+            2. Create repository
+            3. Create credentials
+            4. Create ansible catalog item with provision.yml playbook
+            5. Create ansible catalog item with retire.yml playbook
+            6. Migrate VM from vmware to RHV/OSP using the above catalog items
+    """
+
+    creds = credentials[source_provider.data.templates.get("rhel7_minimal").creds]
```

Comment:
**Requred**

To be safe from `TypeError`, this should be `(...).templates.get('rhel7_minimal', {})['creds']`

Otherwise, this doesn't appear to be a `required_field` when marking/filtering providers, and this call should be wrapped with try/except to call `pytest.fail` or `pytest.skip` when the lookup fails.